mmdet/core/hook/ema.py

```diff
@@ -17,7 +17,7 @@ class BaseEMAHook(Hook):
         momentum (float): The momentum used for updating ema parameter.
             Ema's parameter are updated with the formula:
            `ema_param = (1-momentum) * ema_param + momentum * cur_param`.
-            Defaults to 0.0002.
+            Defaults to 0.0001.
```

Comment:
Changing the default value may cause BC-breaking. Suggest changing this value in config.